docs/arcadia-users-group/20221017-conda/lesson.md

```diff
@@ -0,0 +1,455 @@
+
+**Package managers and why we use them.** 
+A [package manager](https://en.wikipedia.org/wiki/Package_manager) is a software tool that automates the process of installing, upgrading, configuring, or removing software from your computer.
+[Pip](https://pypi.org/project/pip/) (python), [BiocManager](https://cran.r-project.org/web/packages/BiocManager/vignettes/BiocManager.html) (Bioconductor R packages), and [APT](https://en.wikipedia.org/wiki/APT_(software)) (ubuntu) are three commonly encountered package managers. 
+Package managers make installation software easier. 
+
+**Environments and why we use them.** 
+Environment managers address the problems created when software is installed system-wide.
+System-wide installs create complex dependencies between disparate projects that are difficult to entangle to version compute environments and that can create dependcency conflicts. 
+An environment is a directory that contains a specific collection of packages/tools that you have installed. 
+An environment manager is a software tool that organizes where and how software is installed on a computer.
+For example, you may have one environment with Python 2.7 and its dependencies, and another environment with Python 3.4 for legacy testing. 
+If you change one environment, your other environments are not affected. 
+You can easily activate or deactivate environments, which is how you switch between them. 
+
+## What is Conda?
+
+Conda is open source package and runs on Windows, Mac OS and Linux.
+
++ Conda can quickly install, run, and update packages and their dependencies.
++ Conda can create, save, load, and switch between project specific software environments on your local computer.
+
+Conda as a package manager helps you find and install packages. 
+If you need a package that requires a different version of Python, you do not need to switch to a different environment manager, because Conda is also an environment manager. 
+With a few commands, you can set up a totally separate environment to run that different version of Python, while continuing to run a different version of Python in another environment.
+
+## How does Conda work?
+
+<center>
+<figure markdown>
+  ![conda](conda.png){ width="700" }
+  <figcaption> Cartoon of conda environments by Gergely Szerovay <a href='https://www.freecodecamp.org/news/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c/' target='_blank'>www.freecodecamp.org</a> </figcaption>
+</figure>
+</center>
+
+### Installing Conda
+
+<center>
+<figure markdown>
+  ![installingconda](installing_conda.png){ width="700" }
+  <figcaption> Cartoon of decision points for conda installation by Gergely Szerovay <a href='https://www.freecodecamp.org/news/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c/' target='_blank'>www.freecodecamp.org</a> </figcaption>
+</figure>
+</center>
+
+For this lesson, we'll install miniconda. 
```

Comment:
I don't know if it's too much to add here the differences between Anaconda and Miniconda and why we are going with Miniconda

Reply:
I'll add it at the bottom where I talk about the rest of the conda ecosystem and how all of the pieces fit together